client/src/common/organization.ts

```diff
@@ -37,3 +38,21 @@ export function formatExpirationTime(duration: Duration): Translatable {
   }
   return 'HomePage.organizationList.expiration.expired';
 }
+
+export function checkTrialAndSetExpiration(
+  device: AvailableDevice,
+  isTrialOrg: boolean,
+  expirationDuration: Duration,
+): { isTrialOrg: boolean; expirationDuration: Duration } {
+  if (device && expirationDuration) {
```

Comment:
No need to check the device, it's typed as `AvailableDevice`, meaning it will always have a value.